### DISCUSSION TITLE
🌟  Should we limit the number of lines in Card and Tiles Titles

### DISCUSSION THREAD

**Discussion by fraincs** [gsoft-inc/sg-orbit#1114] [Ideas]
## Description

Cards and Tiles titles have no set limits on the number of lines they accept as seen here: 
![image](https://user-images.githubusercontent.com/361632/147116058-26918b2d-334a-41f3-95f8-722b23388fbc.png)

Should we have a limit? Should we do a multi line truncation after two lines?

![image](https://user-images.githubusercontent.com/361632/147116127-363c4dac-83b8-4537-8253-fd1fb123a56f.png)

The design choice should be documented in the Card Usage section, to be.

## Checklist

- [ ] Figma
- [ ] Design respect accessibility specs
- [ ] Usage section is written

**Comment by patricklafrance** [comment]
I believe it should be handled with documentation guidelines rather than code. What do you think?

**Comment by fraincs** [comment]
I agree

**Comment by patricklafrance** [comment]
What's the execution plan then?

**Comment by fraincs** [comment]
Will document with the input of designers and close the issue once done.
